pr_check.sh

```diff
@@ -20,15 +20,17 @@ set -exv
 # Options that must be configured by app owner
 # --------------------------------------------
 APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
-REF_ENV="insights-production"
+# TODO set it to "insights-stage" once we have dvo-writer/dvo-extractor stage deploy config
```

Comment:
this line can be removed

Reply:
discussed on slack: we'll do it in a separate PR